app/saml.server.ts

```diff
@@ -0,0 +1,53 @@
+// saml server based on following PR
+// https://github.com/remix-run/examples/pull/130/files/ec66b3060fac83eec2389eb0c96aad6d8ea4aed1#diff-02d2b71e481b2495b8a72af14f09fc28238298c7f1d19a540e37c9228985b0da
+import * as samlify from 'samlify';
+import * as validator from '@authenio/samlify-node-xmllint';
+import { readFileSync } from 'fs';
+
+samlify.setSchemaValidator(validator);
+
+// Here we configure the service provider: https://samlify.js.org/#/sp-configuration
+
+const sp = samlify.ServiceProvider({
+  entityID: process.env.SAML_ENTITY_ID,
+  nameIDFormat: ['urn:oasis:names:tc:SAML:2.0:nameid-format:persistent'],
+  wantAssertionsSigned: true,
```

Comment:
Is this true?

Reply:
http://localhost:8080/sp

```
<EntityDescriptor xmlns="urn:oasis:names:tc:SAML:2.0:metadata" xmlns:assertion="urn:oasis:names:tc:SAML:2.0:assertion" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" entityID="http://host.docker.internal:8080/sp">
<SPSSODescriptor AuthnRequestsSigned="false" WantAssertionsSigned="true" protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol">
<NameIDFormat>urn:oasis:names:tc:SAML:2.0:nameid-format:persistent</NameIDFormat>
<SingleLogoutService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="http://localhost:8080/logout/callback"/>
<AssertionConsumerService index="0" Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="http://localhost:8080/login/callback"/>
</SPSSODescriptor>
</EntityDescriptor>
```

I think so

Reply:
But that is being generated based on your config.  I'm saying, "Are we sure we want this assertion to be signed"?  Signed with what?  We have no keys in here?  I'm confused.

Reply:
Although I can't properly tell if the assertions are being signed or not, I would think it should say so in the extract but it doesn't seem to. This is something that I am going to spend some more time on.

Reply:
I thought the IDP signs the assertions, the SP is just saying that it wants the signature, and the IDP has its own set of keys and cert